src/Sulu/Bundle/CategoryBundle/Category/CategoryManager.php

```diff
@@ -272,7 +290,10 @@ public function move($id, $parent)
             throw new CategoryIdNotFoundException($parent);
         }
 
+        $previousParentId = $category->getParent()->getId();
```

Comment:
@nnatter think this could also be `null` when it is a root Category moved into a child